x-pack/plugins/ml/public/application/explorer/hooks/use_selected_cells.test.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import moment from 'moment';
+import { renderHook } from '@testing-library/react-hooks';
+import { useSelectedCells } from './use_selected_cells';
+import { ExplorerAppState } from '../../../../common/types/ml_url_generator';
+import { TimefilterContract } from '../../../../../../../src/plugins/data/public';
+
+import { useTimefilter } from '../../contexts/kibana';
+
+jest.mock('../../contexts/kibana');
+
+describe('useSelectedCells', () => {
+  test('should no set state when the cell selection is correct', () => {
```

Comment:
Nit - typo - should be `not` rather than `no`

Reply:
fixed in https://github.com/elastic/kibana/pull/86461/commits/d42a64ef9131eb19bbf6f3cd22b9879bfc7308ed